internal/serve/serve.go

```diff
@@ -74,7 +82,10 @@ type handlerDeps struct {
 	AccountService            services.AccountService
 	AccountSponsorshipService services.AccountSponsorshipService
 	PaymentService            services.PaymentService
-	AppTracker                apptracker.AppTracker
+	// TSS
+	RPCCallerServiceChannel tss.Channel
+	TSSRouter               tssrouter.Router
+	AppTracker              apptracker.AppTracker
```

Comment:
The AppTracker is not a TSS-specific component, right?

Reply:
Changed this so the comment is more specific